cmd/traffic_layout/traffic_layout.cc

```diff
@@ -153,14 +153,14 @@ produce_layout(bool json)
   if (json) {
     printf("{\n");
   }
-  print_var("PREFIX", Layout::get()->prefix, json, false); // Don't free this
+  print_var("PREFIX", ats_strdup(Layout::get()->prefix.c_str()), json, false); // Don't free this
   print_var("BINDIR", RecConfigReadBinDir(), json);
   print_var("SYSCONFDIR", RecConfigReadConfigDir(), json);
-  print_var("LIBDIR", Layout::get()->libdir, json, false); // Don't free this
+  print_var("LIBDIR", ats_strdup(Layout::get()->libdir.c_str()), json, false); // Don't free this
```

Comment:
I agree. I missed this detail. ats_strdup is making new string, we can just pass in the values straight